### PR TITLE
Fixed navbar floating button on mobile and hard coded buy tickets button

### PR DESCRIPTION
**Pull request by AndrewBestbier**

### CODE REVIEW

File: shared/components/NavigationBar/index.scss [hunk 1]
```diff
@@ -91,7 +91,7 @@
     &--mobile {
       position: absolute;
       right: 20px;
-      padding-top: 10px;
+      top: 274px;
```
Comment (lpil) [inline shared/components/NavigationBar/index.scss]: This seems hacky.

Comment (lpil) [inline shared/components/NavigationBar/index.scss]: Oh, it's already absolutely positioned. Still seems hacky :/

**Comment by lpil** [comment]
Yeah go for it

File: shared/components/TicketList/index.js [hunk 2]
```diff
@@ -21,19 +21,28 @@ TicketPrice.propTypes = {
...
 
-export function BuyTickets({ tickets }) {
-  const ticketsAvailable = tickets.some((ticket) => ticket.available);
-  const actionClasses = classnames({
-    'TicketList__booking-btn': true,
-    'TicketList__booking-btn--active': ticketsAvailable,
-    'TicketList__booking-btn--disabled': !ticketsAvailable,
-  });
+export function BuyTickets() {
+  // const ticketsAvailable = tickets.some((ticket) => ticket.available);
```
Comment (asavin) [inline shared/components/TicketList/index.js]: Is there really need for commented code to be committed?

**Pull request merged**